create-base-path-mapping-service.js

```diff
@@ -51,8 +64,21 @@ class CreateBasePathMappingService {
 
 	}
 
+	_deleteBasePathMapping() {
+		return this._getRestApiInfo()
+			.then(apiInfo => {
+				return this._provider.request("APIGateway", "deleteBasePathMapping", {
+					basePath: this._config.basePath,
+					domainName: this._config.domainName
+				});
+			});
+
+	}
+
 	_getRestApiInfo() {
-		return this._provider.request("APIGateway", "getRestApis")
+		return this._provider.request("APIGateway", "getRestApis", {
```

Comment:
This is actually a separate fix to handle case when there are more than 25 apis, which is something we were running into.